tests/README.md

```diff
@@ -3,7 +3,7 @@
 ## Sqlness manual
 
 ### Case file
-Sqlness has three types of file
+Sqlness has two types of file
```

Comment:
nice catch 🤣

Reply:
I took it as a "test" to see I check the docs before doing stuff 😅